lib/services_cli.rb

```diff
@@ -210,33 +186,12 @@ def restart(target, custom_plist = nil, args:)
         if was_run
           run(service)
         else
-          start(service, custom_plist, args: args)
+          start(service, plist_file, verbose)
```

Comment:
I think this might need to be 

```ruby
start(service, plist_file, verbose: verbose)
```

to match the new kwarg signature for this method.